src/templates/Studio/Studio.js

```diff
@@ -62,7 +62,7 @@ export default ({ location, data: { indexPage, studioPage } }) => {
   return (
     <Layout>
       <SEO title="Studio" pathName={studioPage.fields.slug} />
-      <Wonder introRef={introRef} />
+      <Wonder />
```

Comment:
Can the `introRef` variable be completely removed? Does not seem like it's doing anything.

Reply:
Removed